src/uu/pmap/src/pmap.rs

```diff
@@ -49,11 +49,12 @@ pub fn uumain(args: impl uucore::Args) -> UResult<()> {
             }
         }
 
-        match parse_maps(pid) {
-            Ok(total) => println!(" total {total:>16}K"),
-            Err(_) => {
-                set_exit_code(1);
-            }
+        if matches.get_flag(options::DEVICE) {
+            output_device_format(pid).map_err(|_| set_exit_code(1)).ok();
```

Comment:
Yes, the original `pmap` doesn't show an error message if `/proc/<PID>/maps` can't be read:
```
$ pmap 1 --device
1:   /sbin/init \vmlinuz-linux
$ echo $?
1
```
In this example, no memory map is shown because `/proc/1/maps` is not accessible for a normal user.